app/graphql/mutations/create_mutation.rb

```diff
@@ -6,7 +6,6 @@ def resolve(params)
       object = initialize_object(params)
 
       validate_object(object)
-      authorize!(object, :create)
```

Comment:
I don't think removal of authorization is a solution. Non-admin users should have proper permissions to manipulate with the resources. Did you try to run the mutation as a user with `create_hosts` permission assigned?

Reply:
Our users all have the create_hosts permission. But only the ones with Administrator permission can create hosts via GraphQL.

Reply:
I close this PR, it seems we mangled our permission a little bit too much